tests/certbot_dns_snap_test.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
```

Comment:
How much work and boilerplate would we need to move this into `certbot-ci`? I don't think we necessarily have to do it now, but I think it could be nice to have that there in the long term.

Reply:
Not that much. And it gives the opportunity to add some goodies that a reuse from the windows installer integration tests (explicit approval for potential persistent changes, add options to pytest).

Did that.